File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/RangeBarChartsLocalDataViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Kendo.Mvc.Examples.Models
{
    public class RangeBarChartsLocalDataViewModel
    {
        public int FromA { get; set; }
        public int ToA { get; set; }
        public int FromB { get; set; }
        public int ToB { get; set; }
        public string Day { get; set; }

        public RangeBarChartsLocalDataViewModel()
        {
        }

        public RangeBarChartsLocalDataViewModel(int fromA, int toA, int fromB, int toB, string day)
        {
            this.FromA = fromA;
            this.ToA = toA;
            this.FromB = fromB;
            this.ToB = toB;
            this.Day = day;
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/RangeBarChartsLocalDataViewModel.java
package com.kendoui.spring.models;

public class RangeBarChartsLocalDataViewModel {
    public RangeBarChartsLocalDataViewModel(int fa, int ta, int fb, int tb, String day) {
       this.FromA = fa;
       this.ToA = ta;
       this.FromB = fb;
       this.ToB = tb;
       this.Day = day;
    }

    private int FromA;
    private int ToA;
    private int FromB;
    private int ToB;
    private String Day;

    public int getFromA() {
        return FromA;
    }

    public void setFromA(int fromA) {
        FromA = fromA;
    }

    public int getToA() {
        return ToA;
    }

    public void setToA(int toA) {
        ToA = toA;
    }

    public int getToB() {
        return ToB;
    }

    public void setToB(int toB) {
        ToB = toB;
    }

    public int getFromB() {
        return FromB;
    }

    public void setFromB(int fromB) {
        FromB = fromB;
    }

    public String getDay() {
        return Day;
    }

    public void setDay(String day) {
        Day = day;
    }
}
